recipes/libcurl/all/conanfile.py

```diff
@@ -126,6 +130,7 @@ def source(self):
         tools.download("https://curl.haxx.se/ca/cacert.pem", "cacert.pem", verify=True)
 
     def build(self):
+        os.chdir(self.source_folder) 
```

Comment:
this isn't needed I believe

Reply:
it is, if you use the development workflow, and/or specify the source directory via command line. 
I think, it is actually a bug of conan to not automatically change into the source directory, but this should be discussed not here, mabye

Reply:
then pls use `tools.chdir` context manager

Reply:
done, will soo push